### PR TITLE
update the Gemnasium instructions

### PR DESCRIPTION
**Pull request by afeld**
Just went through it with @erik-burgess and @ccostino.

### CODE REVIEW

File: _pages/security/static-analysis.md [hunk 1]
```diff
@@ -16,26 +16,32 @@ There are tools for JS, Ruby, and Python, and you are encouraged to set up this
...
 Code analysis can be run locally with the following open source tools. These tools provide results similar (and in some cases, identical) to the hosted services above.
 
-#### Gemnasium setup
+#### Gemnasium
 
-To set up Gemnasium, you'll need to be invited to the devops@gsa.gov account (ask in [#infrastructure](https://gsa-tts.slack.com/messages/infrastructure/)). In this process, be sure to log in via a password (rather than Github login) as that'll tie access to your gsa.gov email (as opposed to personal email).
+##### Setting up your account
+
+1. [Sign into Gemnasium via GitHub.](https://gemnasium.com/login)
+1. Go to [your settings page](https://gemnasium.com/settings) and change it to use your work email, if it isn't already.
```
Comment (ccostino) [inline _pages/security/static-analysis.md]: Ah nice, so you can just sign in with GitHub?

Comment (afeld) [inline _pages/security/static-analysis.md]: Yep! Unclear if it changes anything, but it works.

Comment (cmc333333) [inline _pages/security/static-analysis.md]: Be careful here -- you don't want to associate the account with your personal email, which was the reason for having a separate password. Perhaps that requirement has changed?

Comment (afeld) [inline _pages/security/static-analysis.md]: @cmc333333 That's why this instruction is telling them to change it...

Comment (cmc333333) [inline _pages/security/static-analysis.md]: :shrug: we must have missed that the first time around. All I remember is being told explicitly to create a separate account and not log in with Github. If we can't figure out a good reasoning for that, agreed that we should remove.

Comment (erik-burgess) [inline _pages/security/static-analysis.md]: There are currently several accounts using personal email addresses. I'll reach out to those folks and ask them to update. Everything else looks fine. The discussion about GH accounts was well before my time here and I'm not seeing any references to it.

**Comment by afeld** [comment]
Anyone care to merge?

**Comment by ccostino** [comment]
I don't see any other objections, thanks @afeld!

**Pull request merged**